install.sh

```diff
@@ -447,7 +447,7 @@ fi
 cd "/usr" || exit 1
 
 ####################################################################### script
-USABLE_GIT="$(command -v git)"
+USABLE_GIT="$(readlink -f "$(command -v git)")"
```

Comment:
Need to only use `readlink` here if it's a symlink otherwise things go pretty weird:

```
$ readlink -f $(/usr/bin/git)
/opt/homebrew/etc
```

Reply:
>```
>readlink -f $(/usr/bin/git)
>```

Doesn't that try to parse the _output_ of `/usr/bin/git` as a pathname?

Reply:
It does. I am apparently a moron 😂.

Ignore me, good to ship!